src/lib/sticky-header/sticky-header.md

```diff
@@ -0,0 +1,57 @@
+The `cdkStickyHeader` directive "sticks" an element to the top of a scrolling container
+while its corresponding `cdkStickyRegion` is in view.
+
+<!-- example(sticky-header-overview) -->
+
+The parent scrolling container for a sticky-header must be marked with `cdkScrollable`.
+This scrolling container contians one or more elements with `cdkStickyRegion`,
+each with their own `cdkStickyHeader`. As the regions are scrolled, the header for the
+region currently within the scrolling viewport will stick to the top.
+
+![alt text][id]
+
+[id]: ./sticky-header.jpg
```

Comment:
We'd need to add the diagram to the assets on the docs site rather than here. Go ahead and leave it out for now and we can add it in a later PR.